ts/features/wallet/component/features/PaymentMethodInitiatives.tsx

```diff
@@ -30,18 +31,23 @@ const PaymentMethodInitiatives = (props: Props): React.ReactElement | null => {
 
   const dispatch = useIODispatch();
 
-  const refresh = React.useCallback(() => {
-    dispatch(
-      idPayInitiativesFromInstrumentRefreshStart({
-        idWallet: idWalletString
-      })
-    );
-    return () => {
-      dispatch(idPayInitiativesFromInstrumentRefreshStop());
-    };
-  }, [idWalletString, dispatch]);
-
-  useFocusEffect(refresh);
+  useFocusEffect(
+    React.useCallback(() => {
+      dispatch(
```

Comment:
I agree with @Hantex9 , why move this call outside of the hook?

Reply:
The function isn't used elsewhere, so I thought it wasn't useful to instantiate it outside the hook.